src/Console/Command/DescribeCommand.php

```diff
@@ -0,0 +1,326 @@
+                '<comment>   ---------- begin diff ----------</comment>%s%%s%s<comment>   ----------- end diff -----------</comment>',
+                PHP_EOL,
+                PHP_EOL
+            ));
+
+            foreach ($description->getCodeSamples() as $index => $codeSample) {
+                $old = $codeSample[0];
+                $tokens = Tokens::fromCode($old);
+                $fixer->configure($codeSample[1]);
+                $fixer->fix(new StdinFileInfo(), $tokens);
+                $new = $tokens->generateCode();
+                $diff = $differ->diff($old, $new);
+
+                if (null === $codeSample[1]) {
+                    $output->writeln(sprintf(' * Example #%d.', $index + 1));
+                } else {
+                    $output->writeln(sprintf(' * Example #%d. Fixing with configuration: <comment>%s</comment>.', $index + 1, $this->arrayToText($codeSample[1])));
+                }
+                $output->writeln($diffFormatter->format($diff, '   %s'));
+                $output->writeln('');
+            }
+        }
+
+        if (!$description) {
+            $output->writeln(sprintf('<question>This rule is not yet described, do you want to help us and describe it?</question>'));
+            $output->writeln('Contribute at <comment>https://github.com/FriendsOfPHP/PHP-CS-Fixer</comment>');
+            $output->writeln('');
+        }
+    }
+
+    /**
+     * @param InputInterface  $input
+     * @param OutputInterface $output
+     * @param string          $name
+     */
+    private function describeSet(InputInterface $input, OutputInterface $output, $name)
+    {
+        if (!in_array($name, $this->getSetNames(), true)) {
+            throw new DescribeNameNotFoundException($name, 'set');
+        }
+
+        $ruleSet = new RuleSet(array($name => true));
+        $rules = $ruleSet->getRules();
+        ksort($rules);
+
+        $fixers = $this->getFixers();
+
+        $output->writeln(sprintf('<info>Description of %s set.</info>', $name));
+        $output->writeln('');
+
+        $help = '';
+
+        foreach ($rules as $rule => $config) {
+            $help .= sprintf(
+                " * <info>%s</info>%s\n   | %s\n%s\n",
+                $rule,
+                $fixers[$rule]->isRisky() ? ' <error>[risky]</error>' : '',
+                $fixers[$rule]->getDescription(),
+                true !== $config ? sprintf("   <comment>| Configuration: %s</comment>\n", $this->arrayToText($config)) : ''
+            );
+        }
+
+        $output->write($help);
+    }
+
+    /**
+     * @param FixerInterface $fixer
+     *
+     * @return bool
+     */
+    private function isFixerConfigurable(FixerInterface $fixer)
+    {
+        try {
+            $fixer->configure(array());
+
+            return true;
+        } catch (UnallowedFixerConfigurationException $e) {
+            return false;
+        } catch (\Exception $e) {
+            return true;
+        }
```

Comment:
yes, i didn't pick up your changes, sorry, this should be extracted and reused instead of copy-pasted first, until that let have the copy-pasted code looks the same in both places.